django/views/generic/base.py

```diff
@@ -152,10 +155,24 @@ def get_template_names(self):
 
 class TemplateView(TemplateResponseMixin, ContextMixin, View):
     """
-    Render a template. Pass keyword arguments from the URLconf to the context.
+    Render a template.
+
+    Deprecated: Pass keyword arguments from the URLconf to the context.
     """
     def get(self, request, *args, **kwargs):
-        context = self.get_context_data(**kwargs)
+        context_kwargs = {}
```

Comment:
Can you put this block in a helper?

```
context_kwargs = _adams_helper_with_a_good_name(...)
```

Reply:
👌 Decided not to put my name in the function signature.